### PR TITLE
mtcp_restart: Fix a bug introduced in 71e1f6c9 for Intel compiler

### PR DESCRIPTION
**Pull request by rohgarg**

### CODE REVIEW

**Comment by gc00** [comment]
The bug was in commit 71e1f6c9.  (Adding this comment to make the link clickable. :-) )

File: src/mtcp/mtcp_restart.c [hunk 1]
```diff
@@ -417,7 +417,7 @@ static void restart_fast_path()
   asm volatile(
    // 96 = offsetof(RestoreInfo, rinfo.restorememoryareas_fptr)
```
Comment (gc00) [inline src/mtcp/mtcp_restart.c]: Please also update the comment:  `// 104 - offsetof(...)`.
Thanks for catching this bug fix of mine.  But I thought I had tested my code and it worked.  Does the restart really fail without this?  If so, thanks again.

Comment (rohgarg) [inline src/mtcp/mtcp_restart.c]: The bug would only appear when compiled with `icc` (Intel compiler).

**Comment by gc00** [comment]
Aha!  It's about icc.  Thanks.  After the minor change in the comment is added, feel free to push this in.  LGTM.

**Pull request merged**